### ISSUE TITLE
Duplicates in included list when selecting larger dataframes

### ISSUE THREAD

**Issue by oddaspa** [clarify/pyclarify#141]
**Describe the bug**
When sending multiple requests to the `data_frame` endpoint with `include=["item"]`, the included items are appended to the list. If the item exists in the list already, a duplicate of the item is added to the list.

**To Reproduce**
```python
from pyclarify import Client
from pyclarify.query import Filter, In

client = Client("./clarify-credentials.json")
response = client.dataframe(
    filter=Filter(fields={"id": In(value="dsadsklajf")}, 
    gte="2020-10-10T00:00:00Z", 
    lt="2022-10-10T00:00:00Z"
)

>>> len(response.result.included.items)
... 12
```

**Expected behavior**
Should be only one item when querying only one item.

**Screenshots**

**System information (please complete the following information):**
OS: MacOS Monterey
Python Version: 3.9
SDK Version: 0.4.1

**Additional context**
Update `__add__()`method to check list before blindly adding. Or filter list at end of request.

**Comment by oddaspa** [comment]
Implementing `__hash__` and `__eq__` methods to item view for set operation at O(1).

**Comment by oddaspa** [comment]
Fixed in #147
